File: src/services/document-management/Myb.document.Model/DocumentModel.cs
using Myb.Common.Models;
using Myb.UserManager.Models;

namespace Myb.document.Model
{
    public class DocumentModel : BaseEntity
    {
        
        public string? DocumentName { get; set; }
       
        public string? CreatedBy { get; set; } 
        public string? EditedBy { get; set; } 

        public DocumentType? DocumentType { get; set; }
        public DocumentStatus? Status { get; set; }
        public long? DocumentSize { get; set; }

        public int? FolderId { get; set; }  
        public virtual Folder? Folder { get; set; }  

        public virtual ICollection<DocumentVersion>? Versions { get; set; }

        public string? file { get; set; }
        /*public string? url { get; set; }*/
        public string? UserId { get; set; }
        public int? CompanyId { get; set; }


    }
}

File: src/front/myb.front/libs/doc-management-module/src/lib/models/DocumentModel.ts
import { BaseModel } from '../../../../shared/infra/models/base.model';
import { DocumentStatus } from './DocumentStatus';
import { DocumentVersion } from './DocumentVersion';
import { Folder } from './Folder';
import { DocumentType } from './DocumentType';




export interface DocumentModel extends BaseModel {
    documentName?: string;
    createdBy: string;
    editedBy: string;
    documentType?: string;
    status?: string;
    documentSize?: number;
    folderId: number | null;
   // folder: Folder | null;
   // versions: DocumentVersion[] | null; 
    // content:string;
    // uploadedFiles?: SelectedFiles[];
    file?: string; 
    url?: string;
   

}